Description: Delay applet startup by some seconds, so that pulseaudio has started up enough to avoid starting it a second time through autospawn.
Index: pulseaudio-2.0/src/daemon/pulseaudio.desktop.in
===================================================================
--- pulseaudio-2.0.orig/src/daemon/pulseaudio.desktop.in	2012-06-08 16:08:04.212943365 +1000
+++ pulseaudio-2.0/src/daemon/pulseaudio.desktop.in	2012-06-08 16:08:40.980943677 +1000
@@ -9,3 +9,4 @@
 NotShowIn=KDE;
 Categories=
 GenericName=
+X-GNOME-Autostart-Delay=8
